packages/components/src/DataViewer/theme.scss

```diff
@@ -0,0 +1,56 @@
+$tc-layout-skeleton-width: 9.6rem !default;
```

Comment:
this is dangerous if it already exists

Reply:
Only used by the DataViewer.stories.js so i think it is not risky. It is a copy of what is done in ui-dataset to use and display viewers components.

Reply:
ok